draft-ietf-ppm-dap.md

```diff
@@ -454,13 +454,17 @@ in the "type" field (within the DAP URN namespace
 
 This list is not exhaustive. The server MAY return errors set to a URI other
 than those defined above. Servers MUST NOT use the DAP URN namespace for errors
-not listed in the appropriate IANA registry (see {{urn-space}}). Clients SHOULD
-display the "detail" field of all errors. The "instance" value MUST be the
-endpoint to which the request was targeted. The problem document MUST also
-include a "taskid" member which contains the associated DAP task ID (this value
-is always known, see {{task-configuration}}), encoded in Base 64 using the URL
-and filename safe alphabet with no padding defined in sections 5 and 3.2 of
-{{!RFC4648}}.
+not listed in the appropriate IANA registry (see {{urn-space}}).
+
+For errors defined in this document, the "title" member MUST be set to the
+Description corresponding to the error type in the table above. The "detail"
```

Comment:
This requirement on `title` doesn't make sense: if the `title` field is a deterministic function of the problem type, then there's no value in including it in problem documents because everyone can just look it up in this table. [RFC 7807 3.1](https://www.rfc-editor.org/rfc/rfc7807#section-3.1) says the title field is "human readable", which I think can be read as "implementation defined" and "not to be relied upon by machines". So I don't think DAP should say anything at all about `title`.

Reply:
Done. I was following @divergentdave recommendation in the issue, but I see his thumbs up here :)